src/util/utils.ts

```diff
@@ -205,9 +209,67 @@ export function joinList(strings_: readonly string[], andOrOr: "and" | "or" = "a
     return joined;
 }
 
+/**
+ * Checks if the running Github server is Enterprise server or not
+ * @returns true if running on GitHub Enterprise Server
+ */
 export function isGhes(): boolean {
     const ghUrl = new URL(
         process.env.GITHUB_SERVER_URL || "https://github.com"
     );
     return ghUrl.hostname.toUpperCase() !== "GITHUB.COM";
 }
+
+/**
+ * The errors messages from octokit HTTP requests can be poor; prepending the status code helps clarify the problem.
+ */
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, @typescript-eslint/explicit-module-boundary-types
+export function getBetterHttpError(err: any): Error {
+    const status = err.status;
+    if (status && err.message) {
+        return new Error(`Received status ${status}: ${err.message}`);
+    }
+    return err;
+}
+
+export function assertNotDefined(value: unknown): asserts value {
```

Comment:
i think this function is named the opposite of what it does?

assertNotDefined -> assertDefined

it would also probably be useful to replace `=== undefined` with `== null`.

Reply:
Oh yes, Will change that.